### ISSUE TITLE
ring.com outdoor camera may have never charged from solar panel

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#652]
Cont'd from https://github.com/130s/30y-130s_life/issues/1203, I've been checking periodically on ring.com app but I've never seen in a camera's "Device Health" section that this camera has charged from solar panel installed.

> [My Ring app is showing a “Not Connected” on the Device Health page (support.ring.com)](https://support.ring.com/hc/en-us/articles/360061517732-Solar-Charger-and-Solar-Panel-for-Ring-Doorbells-troubleshooting)
> Your solar charger or solar panel will only show as connected in the Ring app while it has stored power. Should the device be out of the sun long enough to completely run out of power, it will show up as “Not Connected” in the Ring app. Once the device is exposed to enough sunlight to charge up, the status will return to “Connected.”

Hoping this is the case though, even when in the morning where there's some direct sunlight (e.g. see the img below) status is still not connected. Getting winter morning light for 4 hrs might not be enough?

<img src="https://user-images.githubusercontent.com/1840401/152735071-6b3760c0-6021-4ac5-801f-aeec77e5d6f4.jpg" width=44% />

**Comment by 130s** [comment]
![PXL_20220213_202212501.MP.jpg](https://user-images.githubusercontent.com/1840401/153773463-3d1cb00c-d49f-48df-a061-554818cdc5bb.jpg)
Changed angle. Will see if it charges.

**Comment by 130s** [comment]
It's only 930a but to my eyes the panel is getting a good sunlight. But ring.com app still shows it hasn't detected the charge from the solar panel.
![PXL_20220214_143320068 MP](https://user-images.githubusercontent.com/1840401/153886275-eb06f21a-e0c9-480c-9440-d955f4817fef.jpg)

**Comment by 130s** [comment]
As posted in https://community.ring.com/t/solar-panel-on-spotlight-cams-showing-not-connected/114738/3, I just had to push harder. Now I see ring.com detected the solar panel and the battery level went up.
> I second to @SolarEclipse. I also had to push the power plug of the panel harder until it clicked into the camera. No troubleshooting guides I’ve looking into mentioned this trick.

I also lowered the panel position as the tip of the roof is almost shading at 10am already.
![PXL_20220214_152610453 MP](https://user-images.githubusercontent.com/1840401/153894137-1010dcaa-9e10-44aa-acac-d1e98780afb9.jpg)
